src/main/resources/config/liquibase/changelog/20230619153953_changelog.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.18.xsd">
+    <changeSet author="jakubriegel" id="1687181999663-197">
+        <addColumn tableName="submission">
+            <column name="plagiarism_detected" type="bit" defaultValue="0"/>
```

Comment:
I don't think we should add this to the submission. As we have millions of submissions in Artemis and only a very small amount is affected, it would be better to use the existing logic and create a PlagiarismSubmission which is connected to a Submission with its submissionId

Reply:
Logic added in this fix should let me remain all features any database changes and no additional db operations: https://github.com/ls1intum/Artemis/pull/6924/commits/a9ee4f02e6f64065df980ab88abff1faaa16c8f7#diff-ae9c7ad0977a640f219de748eaf59db9f572456fcf85be491b431a4ff6270e46R110. I'll try it and let you know